src/main/java/com/thirdcc/webapp/service/impl/EventCrewServiceImpl.java

```diff
@@ -74,7 +108,10 @@ public EventCrewDTO save(EventCrewDTO eventCrewDTO) {
     public Optional<EventCrewDTO> findOne(Long id) {
         log.debug("Request to get EventCrew : {}", id);
         return eventCrewRepository.findById(id)
-            .map(eventCrewMapper::toDto);
+            .map(eventCrewMapper::toDto)
+            .map(this::mapUserDetails)
+            .map(this::mapEventName)
+            .map(this::mapUserDetails);
```

Comment:
why here mapUserDetails for second time de?

Reply:
Resolved in dba827f85e1232353deb31e387ebb6ee1242cc45